griptape/tools/rest_api/tool.py

```diff
@@ -24,34 +24,20 @@ class RestApiTool(BaseTool):
         request_body_schema: A JSON schema string describing the request body. Recommended for PUT, POST, and PATCH requests.
         request_query_params_schema: A JSON schema string describing the available query parameters.
         request_path_params_schema: A JSON schema string describing the available path parameters. The schema must describe an array of string values.
-        response_body_schema: A JSON schema string describing the response body.
         request_headers: Headers to include in the requests.
     """
 
     base_url: str = field(kw_only=True)
     path: Optional[str] = field(default=None, kw_only=True)
     description: str = field(kw_only=True)
-    request_path_params_schema: Optional[str] = field(default=None, kw_only=True)
-    request_query_params_schema: Optional[str] = field(default=None, kw_only=True)
-    request_body_schema: Optional[str] = field(default=None, kw_only=True)
-    response_body_schema: Optional[str] = field(default=None, kw_only=True)
```

Comment:
I forgot we had this. Basically having the Tool dictate the LLM's final output? Ew.